### PR TITLE
chore(deps): Included dependency review

### PR DESCRIPTION
**Pull request by naveensrinivasan**
> Dependency Review GitHub Action in your repository to enforce dependency reviews on your pull requests.
> The action scans for vulnerable versions of dependencies introduced by package version changes in pull requests,
> and warns you about the associated security vulnerabilities.
> This gives you better visibility of what's changing in a pull request,
> and helps prevent vulnerabilities being added to your repository.

https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-dependency-review#dependency-review-enforcement
Signed-off-by: naveensrinivasan <172697+naveensrinivasan@users.noreply.github.com>

### CODE REVIEW

**Comment by naveensrinivasan** [comment]
@cpanato 👀

**Review by hectorj2f — APPROVED** [review]

File: .github/workflows/depsreview.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  contents: read
+
+jobs:
+  dependency-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout Repository'
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
+      - name: 'Dependency Review'
+        uses: actions/dependency-review-action@3f943b86c9a289f4e632c632695e2e0898d9d67d
```
Comment (cpanato) [inline .github/workflows/depsreview.yml:28]: can we add a comment that says which tag is in use for each action? Similar we have for other workflows. thanks!

Comment (naveensrinivasan) [reply .github/workflows/depsreview.yml:28]: Like I mentioned in the other ones they become obsolete because dependabot doesn’t update it. 

Let me know.

Comment (cpanato) [reply .github/workflows/depsreview.yml:28]: I know, but I've been updating those from time to time
that is ok to keep as is for now
thanks

File: .github/workflows/depsreview.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#
+# Copyright 2021 The Sigstore Authors.
```
Comment (cpanato) [inline .github/workflows/depsreview.yml:2]: 2022

**Review by cpanato — CHANGES_REQUESTED** [review]

File: .github/workflows/depsreview.yml (hunk 2 above)
Comment (cpanato) [reply .github/workflows/depsreview.yml:2]: a small request

**Comment by naveensrinivasan** [comment]
> @naveensrinivasan please sign the DCO

Done! Thanks

**Review by cpanato — DISMISSED** [review]

File: .github/workflows/depsreview.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+permissions:
+  contents: read
+
+jobs:
+  dependency-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout Repository'
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
```
Comment (cpanato) [inline .github/workflows/depsreview.yml:26]: like you did for the other PR can you add the comment for which tag is being used?
thanks

**Review by cpanato — APPROVED** [review]

**Pull request merged**